mythtv/libs/libmythtv/dbcheck.cpp

```diff
@@ -1970,6 +1970,63 @@ static bool doUpgradeTVDatabaseSchema(void)
     if (dbver == "1301")
     {
         LOG(VB_GENERAL, LOG_CRIT, "Upgrading to MythTV schema version 1302");
+
+        // Adding the default template requires these things.
```

Comment:
Please add comment like this:
        // Adding the default template now requires database table recgroups which was
        // initially introduced in the upgrade to schema version 1320. 
        // The initial database created with a 'clean install" has schema version 1307
        // so upgrading to schema version 1302 is not often done.

Reply:
Done.